src/test/java/org/jenkinsci/plugins/matrixauth/PermissionAdderTest.java

```diff
@@ -0,0 +1,55 @@
+package org.jenkinsci.plugins.matrixauth;
+
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import hudson.model.User;
+import hudson.security.ACL;
+import hudson.security.ACLContext;
+import hudson.security.GlobalMatrixAuthorizationStrategy;
+import hudson.security.HudsonPrivateSecurityRealm;
+import hudson.security.pages.SignupPage;
```

Comment:
Wow, never heard of this before. Once the private realm gets moved into a plugin (?), I guess this should go into a `test-jar`.